### PR TITLE
Algorithm for finding if element is present.

### PR DESCRIPTION
**Pull request by Logan-47**
Algorithm for finding whether the element is present in the list or not.

**Fixes issue:**
no issue fixed
-> New Algorithm  Added.

**Changes:**

New file added in cosmos/code/data_structures/src/list/singly_linked_list/operations/find/ including algorithm for finding whether the element is present in the list or not.

### CODE REVIEW

**Comment by Logan-47** [comment]
@AdiChat  I have added a new algorithm for finding if the element is present in the list or not. Would you please review it and suggest the changes if required?

**Comment by Logan-47** [comment]
@arnavb If you can  specify the voilated part it would be really helpful for me.

File: code/data_structures/src/list/singly_linked_list/operations/find/finding_if_element_is _present_in_the_list.c [hunk 1]
```diff
@@ -0,0 +1,66 @@
+/* Check weather element is present in link list (iterative method)*/
+/* Part of Cosmos by OpenGenus Foundation */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+struct node{                /* defining a structure named node */
```
Comment (arnavb) [inline code/data_structures/src/list/singly_linked_list/operations/find/finding_if_element_is _present_in_the_list.c]: As an example here, the spacing isn't correct for the line above, and you don't need newlines between the data members.

File: code/data_structures/src/list/singly_linked_list/operations/find/finding_if_element_is _present_in_the_list.c [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    int data;                 /* stores data */
+    struct node* next;               /* stores the address of the  next node */
```
Comment (arnavb) [inline code/data_structures/src/list/singly_linked_list/operations/find/finding_if_element_is _present_in_the_list.c]: Comments aren't aligned.

File: code/data_structures/src/list/singly_linked_list/operations/find/finding_if_element_is _present_in_the_list.c [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+};
+
+
+void
+insert( struct node **head , int data )         /* insert an integer to the beginning of the list */
```
Comment (arnavb) [inline code/data_structures/src/list/singly_linked_list/operations/find/finding_if_element_is _present_in_the_list.c]: No spaces in parentheses.

**Comment by Logan-47** [comment]
@arnavb  I have made changes can you please review it again.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**
positive_opengenus

**Pull request merged**